packages/block-editor/src/components/block-list/root-container.js

```diff
@@ -46,7 +46,7 @@ function onDragStart( event ) {
 	}
 }
 
-function RootContainer( { children, className }, ref ) {
+function RootContainer( { children, className, __experimentalUIParts = { hasPopover: true } }, ref ) {
```

Comment:
Rather than setting the default for `__experimentalUIParts` to `{ hasPopover: true }`, how about destructuring `__experimentalUIParts`, and setting its `hasPopover` attribute's default to `true`?

```suggestion
function RootContainer( { children, className, __experimentalUIParts: { hasPopover = true } }, ref ) {
```

(This will also require changing L85.)

Reply:
Alternatively, we could consider passing _only_ `hasPopover` to this component (rather than all of `__experimentalUIParts`). This would be similar to [what we're doing](https://github.com/WordPress/gutenberg/pull/19922/files#diff-f3c7dfbd917e42c14ab7c5d2c21b119eR108) with `hasSelectedUI` in `BlockList`.

```suggestion
function RootContainer( { children, className, hasPopover = true }, ref ) {
```

This might be more consistent with prior art, and thus preferrable.

Reply:
> Alternatively, we could consider passing _only_ `hasPopover` to this component (rather than all of `__experimentalUIParts`). This would be similar to [what we're doing](https://github.com/WordPress/gutenberg/pull/19922/files#diff-f3c7dfbd917e42c14ab7c5d2c21b119eR108) with `hasSelectedUI` in `BlockList`.
> 
> This might be more consistent with prior art, and thus preferrable.

Going with this second suggestion!